src/sourcemaps/sourcemaps-wizard.ts

```diff
@@ -38,9 +38,23 @@ type SupportedTools =
   | 'create-react-app'
   | 'angular';
 
-export async function runSourcemapsWizard(
-  options: WizardOptions,
+export async function runSourcemapsWizardWithTelemetry(
```

Comment:
l: I wonder if we need to rename this (and other places) like this...? Isn't it enough to have `runSourcemapsWizard({ telemetryEnabled: true })` to make this clear?

Reply:
My original intention was to make it clear that this wizard would run with telemetry but you're right, it's way cleaner if the withTelemetry call just happens inside the respective wizard's `run*Wizard` function. I refactored this in https://github.com/getsentry/sentry-wizard/pull/348#:~:text=simplify%20runSourcemapsWizard-,4b55d05,-Merge%20state

For the moment this also has an impact on the Apple wizard, namely that we're passing in the unused `telemetryEnabled` flag. cc @brustolin - are you find with me adjusting the apple wizard slightly? I can do it in a follow-up PR.